tests/shell/pipeline/commands/internal.rs

```diff
@@ -1146,5 +1146,5 @@ fn command_not_found_error_shows_not_found_2() {
             export def --wrapped my-foo [...rest] { foo };
             my-foo
         "#);
-    assert!(actual.err.contains("did you mean"));
+    assert!(actual.err.contains("Command `foo` not found"));
```

Comment:
And `Did you mean` here..

Reply:
Right. I've updated all the tests that I touched.